homeassistant/components/fibaro/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
```

Comment:
Instead of putting the battery level of each device (and armed state) in the extra state attributes, we rather make separate entities out of them since they seem useful enough to automate with and to use in dashboards

Reply:
Yes I see that would be much better. There is no rule to track that, but I would follow your proposal.